Xamarin.Forms.Core/BindingExpression.cs

```diff
@@ -143,11 +143,8 @@ void ApplyCore(object sourceObject, BindableObject target, BindableProperty prop
 
 				if (mode == BindingMode.OneWay || mode == BindingMode.TwoWay)
 				{
-					var inpc = current as INotifyPropertyChanged;
-					if (inpc != null && !ReferenceEquals(current, previous))
-					{
+					if (current is INotifyPropertyChanged inpc && !ReferenceEquals(current, previous))
```

Comment:
:(  

SOON (like, right after we merge to master!)

Reply:
fixed. against my will